printer_mods/Minsekt/2.4/Rear_Umbilical/README.md

```diff
@@ -0,0 +1,14 @@
+# Rear Umbilical Addon
+
+| BOM | Pieces |
+| ------ | ------ |
+| M3 Heatinsert | 4x |
+| M3x10 SCHS | 4x |
+
+![umbilical](https://github.com/Minsekt/moronvods/blob/main/Rear_Umbilical/images/rear_umbilical.png)
+
+since my a/b motor mounts are MJF single pieces i needed a way to add umbilical compatibility without the need to reprint anything. so i designed this.
+its however heavily based on [Selliot79's Design](https://github.com/selliott79/Voron-V2-Umbilical-Mounts/). i just designed it to be a bit smaller, to fit a IGUS CF5.05.18 perfectly and moved the mountings to the rear, so theres space for a [Y Endstop relocation](https://github.com/Minsekt/moronvods/tree/main/Rear_Umbilical/Y_Endstop_Relocation) 
```

Comment:
please replace all links with relative links: ../../selliott79/Voron-V2-Umbilical-Mounts/ for example